Library/ViewModels/RewardCellViewModel.swift

```diff
@@ -213,9 +214,8 @@ RewardCellViewModelOutputs {
       )
       .map(first)
 
-    self.cardViewBackgroundColor = Signal.combineLatest(allGoneAndNotABacker, self.boundStylesProperty.signal)
-      .map(first)
-      .map { $0 ? .ksr_grey_100 : .white }
+    self.cardViewBorderIsVisible = Signal.zip(project, reward)
```

Comment:
Any reason to prefer `zip` over `combineLatest` here? It's probably safe given how these kinds of outputs shouldn't update independently, but I think we generally have moved to use `combineLatest` by default.